tests/test_project.py

```diff
@@ -1146,3 +1155,285 @@ def test_dag(self):
                        [0, 0, 0, 0, 0, 0, 0]]
 
         assert adj == adj_correct
+
+
+# Tests for multiple operation groups or groups with options
+class TestGroupProject(TestProjectBase):
+    project_class = GroupTestProject
```

Comment:
I'd rather extend the original template project than add a second one. All the original tests should also work with a project that has been extended with groups. See also my earlier comment.